lib/common.js

```diff
@@ -154,11 +154,13 @@ function stringifyRequest(options, body) {
   let { port } = options
   // TODO-coverage: Add a test to cover the missing condition, or remove if
   // not reachable.
-  if (!port) port = options.proto == 'https' ? '443' : '80'
+  // The step seems useless considering we don't show the port for standard 80/443 anyway.
+  // The net affect this has is to default the port to 80 when the proto *is not* http. (which probably isn't right)
+  if (!port) port = options.proto === 'https' ? '443' : '80'
 
   if (
-    (options.proto == 'https' && port == '443') ||
-    (options.proto == 'http' && port == '80')
+    (options.proto === 'https' && (port === 443 || port === '443')) ||
+    (options.proto === 'http' && (port === 80 || port === '80'))
```

Comment:
Is it ever acceptable to pass strings to this function, or should we standardize on `port` being a number?

I'm less comfortable with the changes in this file since they're connected to missing coverage lines and while the === is fine for parity, it is not clear that we have use cases for all of these conditions (e.g. string `port`).

Reply:
Coverage doesn't seem to be complaining about this change so I'm fine with it for the time being.